packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -766,12 +1058,11 @@ export const DialogueMatchingPage = ({classes}: {
       )}
 
       <h1>Dialogue Matching</h1>
```

Comment:
I would each section of the page like to a variable, e.g. ```const pageHeader = <h1>...```
Then compose them very readable at the end:
```{pageHeader}
{userTable}
{otherTable}```